website/src/documentation/patterns/ai-platform.md

```diff
@@ -0,0 +1,87 @@
+---
+layout: section
+title: "AI Platform integration patterns"
+section_menu: section-menu/documentation.html
+permalink: /documentation/patterns/ai-platform/
+---
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# AI Platform integration patterns
+
+This page describes common patterns in pipelines with Google AI Platform transforms.
+
+<nav class="language-switcher">
+  <strong>Adapt for:</strong>
+  <ul>
+    <li data-type="language-java">Java SDK</li>
+    <li data-type="language-py" class="active">Python SDK</li>
+  </ul>
+</nav>
+
+## Getting predictions
+
+This section shows how to use a cloud-hosted machine learning model to make predictions about new data using Google Cloud AI Platform Prediction within Beam's pipeline.
+ 
+[tfx_bsl](https://github.com/tensorflow/tfx-bsl) is a library that provides `RunInference` Beam's PTransform. `RunInference` is a PTransform able to perform two types of inference. One of them can use a service endpoint. When using a service endpoint, the transform takes a PCollection of type `tf.train.Example` and, for each element, sends a request to Google Cloud AI Platform Prediction service. The transform produces a PCollection of type `PredictLog` which contains predictions.
+
+Before getting started, deploy a machine learning model to the cloud. The cloud service manages the infrastructure needed to handle prediction requests in both efficient and scalable way. Only Tensorflow models are supported. For more information, see [Exporting a SavedModel for prediction](https://cloud.google.com/ai-platform/prediction/docs/exporting-savedmodel-for-prediction).
```

Comment:
If only TensorFlow models are supported, we should say "deploy a TensorFlow model to the cloud."

Also, does the cloud service to which you deploy the model have to be AI Platform? If so, we should say "deploy a TensorFlow model to AI Platform."

Reply:
Alright, but I'd rather keep this sentence: "only TensorFlow models are supported by the transform" anyway. AI Platform, unlike `RunInference`, doesn't only support TensorFlow models.  It's important to put an emphasis on that.

Reply:
SG - that definitely helps remove the ambiguity

Reply:
The new text says "`RunInference` is able to perform an inference that can use a service endpoint." Is the service endpoint for receiving data or sending requests? It might be helpful to clarify.

Reply:
In this case it's for receiving data. In other words, AI Platform Prediction exposes a service endpoint that can receive data. I'll clarify it.